charts/streams-app/templates/log4j-configmap.yaml

```diff
@@ -0,0 +1 @@
+{{ if .Values.log4jConfig }}
apiVersion: v1
kind: ConfigMap
metadata:
  name: {{ template "streams-app.name" . }}-log4j-configmap
data:
  log4j.properties: |
{{ .Values.log4jConfig | indent 4 }}
{{ end }}
```

Comment:
Is that supposed to be just one line?

Reply:
This is strange. This file has 9 lines on my file system. I will push the file with linux line endings again